msteams-platform/concepts/authentication/authentication.md

```diff
@@ -33,7 +33,7 @@ Enable authentication with SSO or third party OAuth IdPs in your tab app, bot ap
 :::row-end:::
 :::row:::
     :::column span="1":::
-        :::image type="content" source="../../assets/images/authentication/tab-sso-icon.png" alt-text="SSO for tab app" link="../../tabs/how-to/authentication/tab-sso-overview.md":::
+        :::image type="content" source="../../assets/images/authentication/tab-sso-icon.png" alt-text="The screenshot describes about the SSO for tab app." link="../../tabs/how-to/authentication/tab-sso-overview.md":::
```

Comment:
This is not a screenshot.
It doesn't describe anything on its own.
See the topic at https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/authentication/authentication#select-authentication-method  and in alt-text of both the thumbnails, describe what these 2 images showcase. I believe the alt-text can be same for the adjacent images in this case. It really is a comparison that we are drawing.

Reply:
resolved